src/units/zone_loader.rs

```diff
@@ -86,13 +86,10 @@ use crate::zonemaintenance::types::{
 
 #[derive(Debug)]
 pub struct ZoneLoader {
-    /// Addresses and protocols to listen on.
-    pub listen: Vec<ListenAddr>,
-
     /// The zone names and (if primary) corresponding zone file paths to load.
     pub zones: Arc<HashMap<String, String>>,
 
-    /// XFR in per secondary zone: Allow NOTIFY from, and when with a port also request XFR from.
+    /// XFR in per secondary zone.
```

Comment:
I think the old comment was still relevant, the ZM will ignore a NOTIFY that is not permitted IIRC.